### PR TITLE
Updates to JS libraries

### PR DESCRIPTION
**Pull request by townxelliot**
Updated:

* jQuery (`v2 => v3`)
* topojson (`v1 => v3`)
* d3 (`v3 => v5`)

Replaced:

* d3-slider replaced by bootstrap-slider (d3-slider doesn't work with d3 v5)

### CODE REVIEW

File: trademapper/js/trademapper.csv.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ define([
 
 	loadCSVFile: function() {
 		this.resetLoadErrors();
-		this.csvFile = this.fileInputElement[0][0].files[0];
+		this.csvFile = this.fileInputElement._groups[0][0].files[0];
```
Comment (aliceh75) [inline trademapper/js/trademapper.csv.js]: Is this accessing a private member? Is there no other way?

File: trademapper/js/trademapper.mapper.js [hunk 2]
```diff
@@ -116,9 +116,9 @@ define(["d3", "topojson", "worldmap", "disputedareas", "countrycentre"], functio
 	setupZoom: function() {
 		var moduleThis = this,
 		zoomed = function() {
-			var translate = d3.event.translate,
-				scale = d3.event.scale;
-			console.log("target: " + d3.target);
+      var translate = [d3.event.transform.x, d3.event.transform.y];
```
Comment (aliceh75) [inline trademapper/js/trademapper.mapper.js]: Not sure why this looks unindented? Is that an issue with github, or (just checking) a tab/space mixup?

Comment (townxelliot) [reply trademapper/js/trademapper.mapper.js]: Tab/space mixup, I'll fix.

File: trademapper/js/trademapper.csv.js (hunk 1 above)
Comment (townxelliot) [reply trademapper/js/trademapper.csv.js]: Might be possibly to get at it, yes. The issue is that this is a d3 select, so the API is a bit odd and I couldn't find the docs. I'll have another look at it and see if I missed something.

File: trademapper/js/trademapper.mapper.js (hunk 2 above)
Comment (townxelliot) [reply trademapper/js/trademapper.mapper.js]: Fixed.

File: trademapper/js/trademapper.csv.js (hunk 1 above)
Comment (townxelliot) [reply trademapper/js/trademapper.csv.js]: Fixed, used d3.event.target.files instead, which is the right way to do it.

**Pull request merged**